test/uk/gov/hmrc/gform/formtemplate/FormTemplatesControllerRequestHandlerTest.scala

```diff
@@ -344,6 +392,145 @@ class FormTemplatesControllerRequestHandlerTest extends WordSpec with MustMatche
        |  }]
        |}""".stripMargin
 
+  private def validRequestBodyWithPrintSectionAndPagePdfHeaderFooter(
+    authModule: String,
+    identifier: String,
+    serviceId: Option[String] = Some(""""serviceId": "Id",""")) =
+    s"""{
+       |  "_id": "newfield",
+       |  "formName": "Testing section change label tttt",
+       |  "description": "Testing the form change label",
+       |  "languages":["en"],
+       |    "printSection": {
+       |        "page": {
+       |            "title": "Test Title",
+       |            "instructions": "Test Instructions",
+       |            "pdfHeader": "#Test PDF Header",
+       |            "pdfFooter": "##Test PDF Footer"
+       |        },
+       |        "pdf": {
+       |        	"header": "##Test PDF Header",
+       |        	"footer": "##Test PDF Footer",
+       |        	"fieldIds": ["elementA", "elementB"]
+       |        }
+       |    },
```

Comment:
Can we change this so that input json looks like this:
```
{
  "page": {
    "title": "SPBP",
    "instructions": "1. Print a copy of the notification"
  },
  "pdfSummary": {
    "header": "[ANY MARKDOWN TEXT]",
    "footer": "[ANY MARKDOWN TEXT]"
  },
  "pdfNotification": {
    "header": "anything",
    "fieldIds": "${field1},${field2}",
    "footer": "anything"
  }
}
```
Why mix what goes into page and with what goes into pdf?